pkg/capabilities/remote_reportable_user_error.go

```diff
@@ -0,0 +1,43 @@
+package capabilities
+
+import (
+	"strings"
+)
+
+const remoteReportableUserErrorIdentifier = remoteReportableErrorIdentifier + "UserError:"
+
+// RemoteReportableUserError indicates the error is remote reportable and caused by user error.
+type RemoteReportableUserError struct {
+	err *RemoteReportableError
+}
+
+func NewRemoteReportableUserError(err error) *RemoteReportableUserError {
```

Comment:
This name is verbose, however it is probably a good thing that it is explicitly marked as remote reportable so it prompts capability authors to consider this aspect when marking a user error

Reply:
I agree with your justification but on the other hand the "Remote" part is misleading because the error can also be used for local capabilities...

Maybe just "ReportableUserError"?